ITR/data/base_providers.py

```diff
@@ -717,121 +821,91 @@ def project_ei_targets(self, company: ICompanyData, production_bm: pd.Series) ->
                 base_year = target.target_base_year
 
                 # Solve for intensity and absolute
+                model_ei_projections = None
                 if target.target_type == "intensity":
                     # Simple case: the target is in intensity
 
                     # If target is not the first one for this scope, we continue from last year of the previous target
-                    if ei_projection_scopes[scope] is not None:
-                        last_year_data = ei_projection_scopes[scope].projections[-1]
+                    if ei_projection_scopes[scope_name]:
+                        (_, last_ei_year), (_, last_ei_value) = ei_projection_scopes[scope_name].projections[-1] 
                     else:
-                        # Get the intensity data
-                        intensity_data = historic_data.emissions_intensities.__getattribute__(scope)
-                        last_year_data = next((i for i in reversed(intensity_data) if np.isfinite(i.value.magnitude)),
-                                              None)
-
-                    if last_year_data is None:  # No historic data, so no trajectory projections to use either
-                        ei_projection_scopes[scope] = None
-                        continue
-
-                    if base_year > last_year_data.year:
-                        trajectory_ei = projected_intensities.__getattribute__(scope).projections
-                        last_year_data = next((ei for ei in trajectory_ei if ei.year == base_year), None)
-                        warnings.warn(f"Emission intensity at base year for scope {scope} target for company "
-                                      f"{company.company_name} is estimated with trajectory projection.")
-
-                    last_year, value_last_year = last_year_data.year, last_year_data.value
+                        last_ei_year = target.target_base_year
+                        last_ei_value = Q_(target.target_base_year_qty, target.target_base_year_unit)
```

Comment:
this `else` condition lost a few calculation steps.
may i suggest to cover it with UT, to ensure, that we don't lose anything important?